payment/blockchain.py

```diff
@@ -1,82 +1,68 @@
 import contextlib
-import stellar_base
 from typing import List
-from kin.errors import AccountExistsError, AccountNotFoundError
-from kin.stellar.horizon_models import TransactionData
-from kin.sdk import Keypair, ChannelManager, SDK
+from kin.errors import AccountNotFoundError
+from kin.transactions import SimplifiedTransaction
+from kin import KinClient
+from kin.account import KinAccount
+from kin import Keypair
+from kin_base import Keypair as BaseKeypair
 from . import config
 from .models import Payment, Wallet, TransactionRecord
-from .utils import get_network_name
 from .log import get as get_log
-from .errors import ParseError, WalletNotFoundError
+from .errors import WalletNotFoundError
+from .config import STELLAR_ENV, APP_SEEDS
 
 
-log = get_log()
+log = get_log('rq.worker')
 _write_sdks = {}
 
 
-def seed_to_address(seed: str) -> str:
-    """convert seed to address."""
-    return Keypair.from_seed(seed).address().decode()
-
-
-def _init(seed='', channels=[]):
-    kin_sdk = SDK(
-        secret_key=seed,
-        horizon_endpoint_uri=config.STELLAR_HORIZON_URL,
-        network=get_network_name(config.STELLAR_NETWORK),
-        channel_secret_keys=channels,
-        kin_asset=stellar_base.asset.Asset(config.STELLAR_KIN_TOKEN_NAME,
-                                           config.STELLAR_KIN_ISSUER_ADDRESS))
-
-    return kin_sdk
-
-
 class Blockchain(object):
-    read_sdk = _init()
-    asset_code = read_sdk.kin_asset.code
-    asset_issuer = read_sdk.kin_asset.issuer
+    read_sdk = KinClient(STELLAR_ENV)
+    minimum_fee = read_sdk.get_minimum_fee()
 
-    def __init__(self, sdk: SDK, root_address, channel_addresses):
+    def __init__(self, sdk: KinAccount, channel: str):
         self.write_sdk = sdk
-        self.root_address = root_address
-        self.channel_addresses = channel_addresses
+        self.write_sdk.raw_seed = BaseKeypair.from_seed(sdk.keypair.secret_seed).raw_seed()
+        self.root_address = self.write_sdk.keypair.public_address
+        self.channel = channel
+        self.channel_address = Keypair.address_from_seed(channel)
 
-    def create_wallet(self, public_address: str, app_id: str,
-                      initial_xlm_amount: int = config.STELLAR_INITIAL_XLM_AMOUNT) -> None:
```

Comment:
no more app_id?

Reply:
app_id is now a built-in part of the sdk, so the sdk has access to it internally